services/user/Fractal/doc/fractal-service.wit

```diff
@@ -20,10 +23,10 @@ interface types {
     // ambassador, u32(1) << 1;
     record member {
         account: account, // primary key
-        @since(feature = invites)
+        @since(feature = fractal-invites)
         inviter: option<account>,
         // display name, bio, avatar - stored in `sites` service
-        joined-at: timestamp,
+        joined-at: timepointsec,
         titles: u32,
     }
```

Comment:
If this contract handles multiple Fractals and therefore Members *of* Fractals, how is the association between Member and Fractal made? Is there James-visible magic I don't see? I think we're missing a `member.list<fractal>` field.

Reply:
Ah, alternatively (and I'm guessing your intention), we could add `fractal: account` to `record member`. That way `member` indicates a member *in* a fractal, such that there could be numerous `record member` entries for the same chain account, one for each `fractal` they participate with, and
We can get the member list of a fractal by querying the `member` records where `fractal` == <fractal name>.

Reply:
Yep, this
> we could add fractal: account to record member